### PR TITLE
Grammar and spelling fixes

### PR DESCRIPTION
**Pull request by scop**
### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `3.1.x`, `4.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Grammar and spelling fixes.

### CODE REVIEW

File: docs/3.0.0-release-note.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ ce930e7a6 [Jaxrs-cxf] Add bean-level cascaded beanvalidation for pojos (@Valid)
 ff1178ad7 [Java][Spring] fix missing optional query params
 2103fadab Fix package declaration for play-framework
 2c6380c84 fix inner item (list, map) for play framework
-e33b350c8 Fix an issue in Play Framework generator where a CSV is empty and transfered to the controllerImp with an empty item.
+e33b350c8 Fix an issue in Play Framework generator where a CSV is empty and transferred to the controllerImp with an empty item.
```
Comment (wing328) [inline docs/3.0.0-release-note.md]: @scop thanks. I've updated the release note in https://github.com/OpenAPITools/openapi-generator/releases/tag/v3.0.0 accordingly

**Comment by wing328** [comment]
After updating the samples, the Shippable tests passed: https://app.shippable.com/github/OpenAPITools/openapi-generator/runs/862/summary/. I'll file a separate PR to update the samples.

**Comment by scop** [comment]
Yeah, seeing that the samples were already quite outdated at least to some extent, I opted to not touch them here. Thanks for taking care of that.

**Comment by wing328** [comment]
@scop no problem. Thanks again for the PR. Have a nice weekend!

**Pull request merged**